functests/1_performance/netqueues.go

```diff
@@ -357,3 +361,17 @@ func getRandomDevice(devices map[string][]int) string {
 	}
 	return device
 }
+
+func getPerformanceProfile() string {
```

Comment:
We have a helper method that fetches the PP by the node selector, does it not fit good enough?
See - https://github.com/openshift-kni/performance-addon-operators/blob/968c23e1a7216135f9e7a4deb77f8339844b5576/functests/1_performance/hugepages.go#L52